ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -136,36 +136,30 @@ def test_numpy_normal(
     )
 
 
+# permutation
 @handle_frontend_test(
-    fn_tree="numpy.random.geometric",
-    input_dtypes=helpers.get_dtypes("float"),
-    p=st.floats(
-        allow_nan=False,
-        allow_infinity=False,
-        width=32,
-        min_value=9.999999747378752e-06,
-        max_value=0.9999899864196777,
-    ),
-    size=st.tuples(
-        st.integers(min_value=2, max_value=5), st.integers(min_value=2, max_value=5)
```

Comment:
seems like you have overwritten the `geometric` test, you should instead write a new test for the new function :slightly_smiling_face: